src/tools/gxpkg.ss

```diff
@@ -858,6 +1046,20 @@
   (parameterize ((current-output-port (current-error-port)))
     (apply displayln args)))
 
+
+(def (remove-duplicates lst)
```

Comment:
Variants of that function already exist in std/misc/list and/or a srfi/1

Reply:
srfi/1 is O(n^2), this is optimized ro preserve order.